sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -458,6 +458,7 @@ private void removeAllStoredErrorLogFiles(UUID id) {
     private void removeStoredThrowable(UUID id) {
         mErrorReportCache.remove(id);
         ErrorLogHelper.removeStoredThrowableFile(id);
```

Comment:
Should be done after wrapper delete so that this step is replayed in case of process crash/exit between the two lines. That would avoid a file leak.